.github/workflows/main.yml

```diff
@@ -43,7 +43,16 @@ jobs:
         with:
           submodules-ssh-key: ${{ secrets.PRIVATE_GITHUB_DEPLOY_TOKEN }}
 
-      - uses: ./.github/actions/ci-setup
+      - uses: pnpm/action-setup@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: pnpm
+
+      - name: Pnpm install
+        run: |
+          pnpm install
```

Comment:
why name?

Reply:
Why not. When played changed this script few times. To not have something like below.  Doesn't matter anyway.

<img width="743" alt="image" src="https://github.com/user-attachments/assets/2991413d-00c6-4585-9742-6919a3aac4af" />